openstack/tools/openstack_regression_tests_runner.sh

```diff
@@ -21,6 +22,10 @@ OPTIONS:
     --func-test-pr PR_ID
         Provides similar functionality to Func-Test-Pr in commit message. Set
         to zaza-openstack-tests Pull Request ID.
+    --rerun configure|test
+        Re-run the charmed-openstack-tester configure or tests phase. This is
```

Comment:
nit: s/tests/test just so it matches the allowed arguments

Reply:
ack yeah i saw that too, will fix